### ISSUE TITLE
ncmat2endf script

### ISSUE THREAD

**Issue by marquezj** [mctools/ncrystal#183]
I have been working on a script to generate thermal scattering files in the ENDF-6 format. This is a common request from users in the radiation transport community that use codes that do not have native support for NCrystal. It replaces [NJOY+NCrystal](https://doi.org/10.1016/j.nima.2021.166227) by extracting and formatting the information directly from the Info objects. The script is available here:

https://github.com/marquezj/ncmat2endf

It uses [endf-parserpy](https://endf-parserpy.readthedocs.io/en/latest/) to generate the files.

I am opening this issue to start the discussion. Later I can generate a pull request to actually incorporate it.

**Comment by tkittel** [comment]
@marquezj a heads up, one of the things needed for this tool is automated tests. Some can use larger files, but we also need some where files and output are of a rather reasonable size so it can be included in the soon-to-be-in-ncrystal-repo-itself tests.

**Comment by tkittel** [comment]
Related to this:

https://github.com/conda-forge/staged-recipes/pull/29368
https://github.com/IAEA-NDS/endf-parserpy/issues/9

**Comment by marquezj** [comment]
I think we can move this to #267 and close it now that #239 is finished.

**Comment by tkittel** [comment]
No, we still have to add the items above to the new issues. Also, 4.2.0 is not out, I don't normally close issues before the related deliverables are in a release.

**Comment by tkittel** [comment]
NCrystal 4.2.0 is out with this - closing!
